### ISSUE TITLE
[a11y] Issue 9: Missing level-one heading (Participation Page)

### ISSUE THREAD

**Issue by NewJerseyStyle** [NewJerseyStyle/polis#67]
**Description:**

The participation page does not contain a level-one heading (`<h1>`).  This heading is important for screen reader users to navigate directly to the main content of the page.

**Steps to Reproduce:**

1. Navigate to the participation page (e.g. `https://pol.is/4cvkai2ctw`)
2. Inspect the page's heading structure.

**Expected Behavior:**

A `<h1>` element should be present to indicate the main content of the page.

**Actual Behavior:**

A `<h1>` element is missing.

**Component/Area:** Participation page

**Source:** Auto evaluation

**Affected Elements:** h1

**Comment by thomassth** [comment]
Should be fixed with https://github.com/CivicTechTO/polis/pull/32

**Comment by NewJerseyStyle** [comment]
See no `<h1>`, assume not fixed

**Comment by YumiChen** [comment]
In test environment the `h1` tag is present. I think #32 has fixed the issue

http://20.83.181.230/9xthj52ztx
![Image](https://github.com/user-attachments/assets/ef9f45ff-489a-4d0c-9d6e-eb20d5f62114)

**Comment by NewJerseyStyle** [comment]
🎉
